lib/containers/common.pm

```diff
@@ -28,32 +28,48 @@ our @EXPORT = qw(install_podman_when_needed install_docker_when_needed allow_sel
   test_container_runtime test_container_image scc_apply_docker_image_credentials scc_restore_docker_image_credentials);
 
 sub install_podman_when_needed {
+    my $host_os = shift;
+    record_info $host_os;
```

Comment:
```suggestion
    my $host_os = shift //= 'sle';
    record_info('HOST', $host_os);
```

Reply:
I forgot to remove the recored_info. But the default is not need it here. the default is sle/opensue which it will run when it reach the `else`